beluga_amcl/package.xml

```diff
@@ -12,29 +12,41 @@
 
   <license>Apache License 2.0</license>
 
-  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 2">ament_cmake</buildtool_depend>
 
   <depend>beluga</depend>
-  <depend>bondcpp</depend>
+  <depend condition="$ROS_VERSION == 2">bondcpp</depend>
```

Comment:
nit^n: Move this line to the depend ROS_VERSION 2 group.

Reply:
Done in eefccf217f66366a53203f9097052ffdc8232003.